frontend/src/app/page.tsx

```diff
@@ -29,6 +30,7 @@ export const metadata: Metadata = {
 
 export default async function Home() {
   const vendors = await getCachedVendors();
+  const userFavorites = await getFavoriteVendorIds();
```

Comment:
does this work for multiple users? this page is rendered server side, so I wonder if the userFavorites has to happen on the client side so that it will be different for each user. I could be wrong though

Reply:
ohhh such a good point !!!!!!!!